packages/react/src/components/ReadOnlyValue/_read-only-value.scss

```diff
@@ -0,0 +1,44 @@
+.#{$iot-prefix}--read-only-value {
+  display: block;
+  margin-bottom: $spacing-06;
+  label,
+  input {
+    display: block;
```

Comment:
There is an outline on focus-visible that we might want to remove:
![image](https://user-images.githubusercontent.com/5167091/159258642-e1ee627c-0d4a-4b96-ad92-5d5495810916.png)

```
&:focus-visible {
    outline: unset;
}
```